src/GafferImage/ImageWriter.cpp

```diff
@@ -706,10 +711,24 @@ ImageWriter::ImageWriter( const std::string &name )
 	addChild( new ImagePlug( "in" ) );
 	addChild( new StringPlug( "fileName" ) );
 	addChild( new StringPlug( "channels", Gaffer::Plug::In, "*" ) );
+
+	ColorSpacePtr colorSpace = new ColorSpace( "__colorSpace" );
```

Comment:
I wonder, will it be necessary to promote the `ColorSpace::contextPlug()` here for some workflows?